README.md

```diff
@@ -81,22 +70,57 @@ with sycl_context("gpu"):
     clustering = DBSCAN(eps=3, min_samples=2).fit(X)
 ```
 
-daal4py API, allows you to use wider set of Intel(R) oneAPI Data Analytics Library algorithms in just one line:
-```py
-import daal4py as d4p
-init = d4p.kmeans_init(data, 10, t_method="plusPlusDense")
-result = init.compute(X)
+# 🛠 Installation
+daal4py is available at the [Python Package Index](https://pypi.org/project/daal4py/),
+on Anaconda Cloud in [Conda-Forge channel](https://anaconda.org/conda-forge/daal4py)
+and in [Intel channel](https://anaconda.org/intel/daal4py).
+
+```bash
+# PyPi
+pip install daal4py
 ```
-You can even run this on a cluster by making simple code changes:
-```py
-import daal4py as d4p
-d4p.daalinit()
-d4p.kmeans_init(data, 10, t_method="plusPlusDense", distributed=True)
-result = init.compute(X, daal4py.my_procid())
-d4p.daalfini()
+
+```bash
+# Anaconda Cloud from Conda-Forge channel (recommended for conda users by default)
+conda install daal4py -c conda-forge
+```
+
+```bash
+# Anaconda Cloud from Intel channel (recommended for conda users who use other components from Intel Distribution for Python)
+conda install daal4py -c intel
 ```
 
-# Scikit-learn patching
+<details><summary>[Click to expand] ℹ️ Supported configurations </summary>
```

Comment:
I'd suggest adding a header before this line: `## Supported configurations` it'll make them more visible